apps/web/controllers/pets/edit.rb

```diff
@@ -7,6 +7,7 @@ class Edit
     expose :pet
 
     def call(params)
+      # TODO How I can apply more security here?
       @pet = PetRepository.find(params[:id])
```

Comment:
Looks good.

`PetRepository` could become big, couldn't it? Maybe PetRepository could delegate the complexity to specific query objetcts, but for now, it is nice like that to me :)